### PR TITLE
INT-1752 Integration 3

### PR DESCRIPTION
**Pull request by hbjORbj**
- call `https://telemetry.intuita.io/sourceControl/github/issues` API request with title, body and repo
- after `html_url` is obtained, redirect user to the GH issue url
- type checking for API response

### CODE REVIEW

File: src/components/webview/SourceControlPanelProvider.ts [hunk 1]
```diff
@@ -209,8 +211,33 @@ export class SourceControlPanelProvider {
...
+						const { title, body } = message.data;
 						// call API to create Github Issue
+						const codemodRegistryRepo =
+							'https://github.com/intuita-inc/codemod-registry';
+						const { data } = await axios.post(
+							'https://telemetry.intuita.io/sourceControl/github/issues',
+							{
+								title,
+								body,
+								userId: storedClerkToken,
```
Comment (grzpab) [inline src/components/webview/SourceControlPanelProvider.ts:223]: hmm, is it a token, or an user ID? this shouldn't be confused.

Comment (hbjORbj) [reply src/components/webview/SourceControlPanelProvider.ts:223]: token. will change in studio backend

Comment (hbjORbj) [reply src/components/webview/SourceControlPanelProvider.ts:223]: https://github.com/intuita-inc/studio-backend/pull/46

Comment (hbjORbj) [reply src/components/webview/SourceControlPanelProvider.ts]: renamed it to `accessToken`. Ref: https://github.com/intuita-inc/studio-backend/pull/47

**Pull request merged**